modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/util/PowwowUtil.java

```diff
@@ -546,6 +547,10 @@ private static PowwowSubscriptionSender _getSubcriptionSender(
 		powwowSubscriptionSender.setScopeGroupId(powwowMeeting.getGroupId());
 		powwowSubscriptionSender.setServiceContext(serviceContext);
 		powwowSubscriptionSender.setUserId(powwowMeeting.getUserId());
+		powwowSubscriptionSender.setNotificationType(PowwowParticipantConstants.STATUS_INVITED);
```

Comment:
why do we need to set notification type to `PowwowParticipantConstants.STATUS_INVITED`?